src/main/java/org/folio/search/configuration/properties/FolioEnvironment.java

```diff
@@ -10,10 +14,20 @@
   private final String okapiUrl;
   private final String env;
 
-  public FolioEnvironment(@Value("${okapi.url}") String okapiUrl,
-    @Value("${env:folio}") String env) {
-
+  public FolioEnvironment(@Value("${okapi.url}") String okapiUrl) {
```

Comment:
can we mover okapi.url to the configuration properties like OkapiConfigurationProperties